version.py

```diff
@@ -1,17 +1,29 @@
 import os
+import re
 
-ZULIP_VERSION = "9.0-dev+git"
+ZULIP_BASE_VERSION = "9.0-dev+git"
```

Comment:
`tools/release` tries to parse this out, so will need to be updated for this change.